packages/event-consumer/src/consumerService.ts

```diff
@@ -0,0 +1,277 @@
+import { match } from "ts-pattern";
+import { Collection, MongoClient } from "mongodb";
+import { logger } from "pagopa-interop-commons";
+import { EService } from "pagopa-interop-models";
+import { EventEnvelope } from "./model/models.js";
+import { config } from "./utilities/config.js";
+import {
+  fromDescriptorV1,
+  fromDocumentV1,
+  fromEServiceV1,
+} from "./model/converter.js";
+
+const {
+  readModelDbUsername: username,
+  readModelDbPassword: password,
+  readModelDbHost: host,
+  readModelDbPort: port,
+  readModelDbName: database,
+} = config;
+
+const mongoDBConectionURI = `mongodb://${username}:${password}@${host}:${port}`;
+const client = new MongoClient(mongoDBConectionURI);
+
+const db = client.db(database);
+const eservices: Collection<{
+  data: EService | undefined;
+  metadata: { version: number };
+}> = db.collection("eservices", { ignoreUndefined: true });
+
+export async function handleMessage(message: EventEnvelope): Promise<void> {
+  logger.info(message);
```

Comment:
Is this a leftover or is there for debugging purposes?

Reply:
It's not a leftover. It's very useful to have the incoming message logged but maybe it's better to use `logger.debug` instead of `logger.info`